metadata/org.liquibase/liquibase-core/4.17.0/resource-config.json

```diff
@@ -16,6 +16,9 @@
         "pattern": "\\Qliquibase.build.properties\\E"
       },
       {
+        "condition": {
+          "typeReachable": "iquibase.parser.core.xml.LiquibaseEntityResolver"
```

Comment:
`iquibase` looks like typo

Reply:
Fixed that in b6b77481d43aea58a63fe711ccc04573686fd13c